docs/semgrep-code/notifications.md

```diff
@@ -56,15 +56,149 @@ To view, add, remove, disable, or enable your saved channels:
 
-<ProcedureIntegrateSlack />
+<!-- <ProcedureIntegrateSlack /> -->
+
+The Semgrep Slack app enables Semgrep Cloud Platform to notify you of new findings after every scan. By receiving notifications within your Slack workspace, developers and security engineers are able to see findings without leaving their current environment. This can lessen the friction between detecting a finding, triaging it, and resolving it.
+
+You can select which channels in your Slack workspace will receive findings. You can also choose to receive findings only for certain repositories (projects) or certain **policies**. 
+
+**Policies **typically correspond to the different Rule board columns. For example, you can choose to receive notifications only for findings generated by rules from the Rule Board's Blocking column.
+
+:::info PREREQUISITE
+You must be a Slack **Workspace Owner** to set up the Semgrep Slack app. 
+:::
+
+#### Installing Semgrep Slack App
+
+To install the Semgrep Slack app, follow these steps:
+
+1. Sign in to your [Semgrep Cloud Platform](https://semgrep.dev/login) account, and then go to **Settings** > **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)**.
+2. On the **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)** page, click **Add Integration** (or **Setup First Integration** if this is your first integration), and then select **Slack**.
+3. Click **Allow**.
+
+#### Setting up notifications for findings in Slack
+
+To set up or subscribe to notifications for findings in your Slack workspace, perform the following steps:
+
+1. In your Slack workspace, find or create a channel for Semgrep notifications.
+2. In the selected Slack channel, enter the following slash command: `/semgrep_subscribe`.
+3. **Optional**: Enter the name of a specific project after `/semgrep_subscribe` to receive findings for that specific project only, for example, `/semgrep_subscribe acme-corp/vulnerable-repo`. The project must be entered in the following format:
+    <br />
+    <code>/semgrep_subscribe <span className="placeholder">ACCOUNT_NAME/REPOSITORY_NAME</span></code>
+4. Choose an organization in the list under **Select target organization**. The dialog box expands with additional options.
+    ![Semgrep Slack app dialog box for subscribing to notifications](/img/semgrep-slack-subscribe.png "Semgrep Slack app dialog box for subscribing to notifications")
+5. **Optional**: Set up additional filters.
+    1. For Semgrep users that receive both Semgrep Code findings and Semgrep Supply Chain vulnerabilities, you can select **target scan types** to subscribe to either Semgrep Code, Semgrep Supply Chain, or both.
+    2. Select any number of policies to receive findings for under the **Selected Policies** field. By default, you are subscribed to all policies, including the Monitor policy. This can potentially result in a noisy message.
```

Comment:
Optional (matter of style we did not define previously, feel free to dismiss). See [style guide](https://developers.google.com/style/procedures). To make the list more concise, you don't have to use indentation:
```suggestion
5. Optional: For Semgrep users that receive both Semgrep Code findings and Semgrep Supply Chain vulnerabilities, you can select **target scan types** to subscribe to either Semgrep Code, Semgrep Supply Chain, or both.
6. Optional: Select any number of policies to receive findings for under the **Selected Policies** field. By default, you are subscribed to all policies, including the Monitor policy. This can potentially result in a noisy message.
```

Reply:
I tried both in an earlier commit and chose this because I want to make it clear we're applying filters. Also having so many "optional" texts clutters up the procedure. I'll take out the bold formatting though.